INFO:root:called-params config/feature_extraction.yaml
INFO:root:loaded params...
INFO:root:{'num_workers': 32, 'base_dir': '/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/', 'pretrained_path': 'pretrained_models/vit_base_patch14_reg4_dinov2_lvd142m_pc24_onlyclassifier_then_all/model_best.pth.tar', 'data': {'class_mapping': 'pretrained_models/class_mapping.txt', 'species_mapping': 'pretrained_models/species_id_to_name.txt', 'train_data': '/home/rtcalumby/adam/luciano/PlantCLEF2025/images_max_side_800/', 'output_dir': '/home/rtcalumby/adam/luciano/PlantCLEF2025/cached_features/'}}
INFO:root:Running... (rank: 0/8)
INFO:root:Initialized (rank/world-size) 0/8
Process Process-3:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
INFO:timm.models._helpers:Loaded state_dict_ema from checkpoint 'pretrained_models/vit_base_patch14_reg4_dinov2_lvd142m_pc24_onlyclassifier_then_all/model_best.pth.tar'
Process Process-2:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
Process Process-5:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
Process Process-8:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
Process Process-4:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
Process Process-7:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
Process Process-6:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
Process Process-1:
Traceback (most recent call last):
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/rtcalumby/miniconda3/envs/fgdcc/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/main_feature_extraction.py", line 44, in process_main
    app_main(args=params)
  File "/home/rtcalumby/adam/luciano/PlantCLEF2025/PlantCLEF2025/engine_feature_extraction.py", line 70, in main
    train_dataset, train_dataloader, dist_sampler = build_train_dataset(image_folder=train_dir,
TypeError: build_train_dataset() missing 2 required positional arguments: 'world_size' and 'rank'
